internal/backend/container.go

```diff
@@ -61,12 +63,28 @@ func (w *NerdctlWrapper) CreateContainer(ctx context.Context, args []string, net
 	return container.Create(ctx, w.clientWrapper.client, args, netManager, options)
 }
 
-func (w *NerdctlWrapper) InspectContainer(ctx context.Context, c containerd.Container) (*dockercompat.Container, error) {
+func (w *NerdctlWrapper) InspectContainer(ctx context.Context, c containerd.Container, sizeFlag bool) (*dockercompat.Container, error) {
 	n, err := containerinspector.Inspect(ctx, c)
```

Comment:
Why are we not using https://github.com/containerd/nerdctl/blob/main/pkg/cmd/container/inspect.go#L38 api which already has support for Size?

Reply:
The linked inspect function prints the inspect response to a provided ioWriter, nerdctl cli uses that. We can't consume it without making upstream changes